### ISSUE TITLE
Impossible to update addon to version 0.1.14

### ISSUE THREAD

**Issue by nepozs** [sviete/ais-ha-addon-cloudflared#18]
### The problem

```
Rejestrator: homeassistant.components.hassio
Źródło: components/hassio/websocket_api.py:135
integracja: Home Assistant Supervisor (dokumentacja, Problemy)
Pierwsze zdarzenie: 16:15:13 (1 zdarzenia)
Ostatnio zalogowany: 16:15:13

Failed to to call /store/addons/6e457389_cloudflared/update - Can't install ghcr.io/sviete/cloudflared/amd64:0.1.14: 500 Server Error for http+docker://localhost/v1.47/images/create?tag=0.1.14&fromImage=ghcr.io%2Fsviete%2Fcloudflared%2Famd64&platform=linux%2Famd64: Internal Server Error ("manifest unknown")
```

### What version of Cloudflared has the issue?

amd64:0.1.13

### What was the last working version of Cloudflared?

amd64:0.1.14

### What type of installation are you running?

Home Assistant OS

### Add-on YAML Configuration

```yaml
NA
```

### Anything in the logs that might be useful for us?

```txt
2024-10-22 16:15:13.904 ERROR (MainThread) [supervisor.docker.interface] Can't install ghcr.io/sviete/cloudflared/amd64:0.1.14: 500 Server Error for http+docker://localhost/v1.47/images/create?tag=0.1.14&fromImage=ghcr.io%2Fsviete%2Fcloudflared%2Famd64&platform=linux%2Famd64: Internal Server Error ("manifest unknown")
```
```


### Steps to reproduce the issue

Update addon to current version

### Additional information

Version 0.1.13 now doesn't work, but for other reasons.

edit
https://github.com/sviete/ais-ha-addon-cloudflared/pkgs/container/cloudflared%2Famd64
image tagged 0.1.14 just do not exist for any supported platform
https://github.com/orgs/sviete/packages?repo_name=ais-ha-addon-cloudflared

**Comment by lyczko** [comment]
same problem here

**Comment by nepozs** [comment]
Fixed with v0.1.15
https://github.com/sviete/ais-ha-addon-cloudflared/releases/tag/v0.1.15
